### DISCUSSION TITLE
Request time out error while doing STT transcription

### DISCUSSION THREAD

**Discussion by samueljazzjohn** [deepgram/community#258] [General help]
### Which Deepgram product are you using?

None

### Details

While doing the STT conversion on real time audio chunk , some time it is showing the request time out error. Can you please help me to find the root cause of that?

<img width="884" alt="Screenshot 2023-07-05 at 11 37 29 AM" src="https://github.com/deepgram/community/assets/64126806/cb345cf7-a656-4508-9467-aa2893cc343d">

### If possible, please attach your code or paste it into the text box.

This is the code snippet I am using
<img width="1044" alt="Screenshot 2023-07-05 at 11 41 52 AM" src="https://github.com/deepgram/community/assets/64126806/1fe852fe-0d20-4524-beee-39c4ad32f906">

**Comment by DamienDeepgram** [comment]
Is it possible that you are not sending audio for more than 10 seconds?

See: https://developers.deepgram.com/docs/recovering-from-connection-errors-and-timeouts-when-live-streaming-audio

If so you might need to send us keep alive messages to keep the connection open

See: https://developers.deepgram.com/reference/streaming#stream-keepalive
